src/base_circuitpython/displayio/group.py

```diff
@@ -32,19 +39,50 @@ def append(self, item):
             raise ValueError(CONSTANTS.LAYER_ALREADY_IN_GROUP)
 
         self.__contents.append(item)
-        item.in_group = True
-        if self.auto_write:
-            self.draw(show=True)
+        item.parent = self
+
+        self.__elem_changed()
+
+    def __elem_changed(self):
+        # ensure that this group is what the board is currently showing
+        # otherwise, don't bother to draw it
+        if (
+            self.__auto_write
+            and self.__check_active_group_ref
+            and board.DISPLAY.active_group == self
+        ):
+            self.draw()
+
+        elif self.in_group:
+
+            # if a sub-group is modified,
+            # propagate to top level to
+            # see if one of the parents are the
+            # current active group

```

Comment:
_&ast;nit&ast;_
Perhaps reorder the comment so it fits in less lines?
Also, if I remember correctly Brett mentioned the advantages of writing comments like text (Capitalize words, add fullstop at the end of a sentence, etc.). Perhaps consider that in the new comments?